src/app/test/db/mongo/util_test.py

```diff
@@ -19,17 +23,24 @@ def model_mocks(monkeypatch):
     role_mock = Mock()
     job_mock = Mock()
     principal_mock = Mock()
+    black_list_mock = Mock()
```

Comment:
I know you modeled this on the other tests here, but I don't know that there's a compelling reason to use mocks here.  Especially for the Event model.  Try removing the mocks and working with actually model instances.  I suspect you won't run into any issues, but if you do we can work through them.